pallets/funding/src/lib.rs

```diff
@@ -1618,48 +1629,128 @@ pub mod pallet {
 		fn on_idle(_now: BlockNumberFor<T>, max_weight: Weight) -> Weight {
 			let mut remaining_weight = max_weight;
 
-			let projects_needing_cleanup = ProjectsDetails::<T>::iter()
-				.filter_map(|(project_id, info)| match info.cleanup {
-					cleaner if <Cleaner as DoRemainingOperation<T>>::has_remaining_operations(&cleaner) =>
-						Some((project_id, cleaner)),
-					_ => None,
-				})
-				.collect::<Vec<_>>();
-
-			let projects_amount = projects_needing_cleanup.len() as u64;
-			if projects_amount == 0 {
-				return max_weight;
-			}
-
-			let mut max_weight_per_project = remaining_weight.saturating_div(projects_amount);
+			// We want at least 5% of max block weight
```

Comment:
Are guesstimating a single participation settlement will occupy max 5%?

Reply:
Its an Guesstimation for now, after creating/running benchmarks we will know more and can adjust this value.